test/d_do_test.d

```diff
@@ -27,15 +30,16 @@ void usage()
           ~ "   relevant environment variables:\n"
           ~ "      ARGS:          set to execute all combinations of\n"
           ~ "      REQUIRED_ARGS: arguments always passed to the compiler\n"
-          ~ "      DMD:           compiler to use, ex: ../src/dmd\n"
+          ~ "      DMD:           compiler to use, ex: ../src/dmd (required)\n"
```

Comment:
`../src/dmd` no longer exists ;-)

Reply:
Yes I believe you just recently changed that because of the `ddmd` to `dmd` change :)  A nice change by the way.